app/assets/javascripts/student_profile/record_service.js

```diff
@@ -197,7 +197,7 @@
           onClick: this.onClickCancel
         }, 'Cancel'),
         (this.props.requestState === 'pending') ? dom.span({}, 'Saving...') : null,
-        (this.props.requestState === 'error') ? dom.span({}, 'Try again!') : null
+        dom.span({}, this.props.requestState)
```

Comment:
Okay, given that there are only two states, I can just update the else part of the 'pending' line.